addons/packages/app-toolkit/0.2.0/bundle/config/developer-namespace.yaml

```diff
@@ -0,0 +1,48 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+
+#@ if/end data.values.developer_namespace != "default":
+---
+apiVersion: v1
+kind: Namespace
+metadata:
```

Comment:
other tools may expect to be able to create a namespace or add a label/annotation to define extra behavior. `kapp` will fail if the namespace already exists, and will remove any other labels/annotation a user expects to be able to place on the resource.

Reply:
**Notes from Zoom Call:**
This could make it tricky if the user wants the namespace created by another tool. By specifying it here, we give `kapp` ownership meaning it will consistently reconcile it to this state making it impossible for other tools to modify it. Suggested fix is to add a flag that disables namespace creation in App Toolkit.

We predict App Toolkit won't be used any time soon for anything more than a playground, so we can cross this bridge when we get feedback that it is bothersome.